src/ImageSharp/Formats/Jpeg/JpegMetadata.cs

```diff
@@ -101,6 +103,11 @@ public int Quality
     /// </remarks>
     public bool? Progressive { get; internal set; }
 
+    /// <summary>
+    /// Gets the comments.
+    /// </summary>
+    public ICollection<Memory<char>>? Comments { get; }
```

Comment:
I don't think this should be nullable. Collections should always have a backing value in a publi API. I would use `IList` also to prevent all the casting and maintain consistency with the `PngMetadata.TextDataProperty`

On that note we could use a specific struct `JpegComData` which would contain the backing memory and can have static `FromString(string value)`

```c#
public readonly struct JpegComData
{
	public static JpegComData FromString(string value) => new(value.AsMemory());

	public JpegComData(ReadOnlyMemory<char> value) 
		=> this.Value = value;
	
	
	public ReadOnlyMemory<char> Value { get; }

	public override string ToString() => this.Value.ToString();
}
```

This would also remove the need for any extension methods.

Reply:
@br3aker What do you think?

Reply:
Hi!
I'm in favor of an empty collection instead of a null one - it's better from a user perspective.
Not so sure about `JpegComData`, do we really need to expose comments as `ReadOnlyMemory<char>` instead of a plain string?

Reply:
I thought we couldn't guarantee that the data was always an ASCII string and that's why we had a collection of char?

Reply:
> I thought we couldn't guarantee that the data was always an ASCII string and that's why we had a collection of char?

True, if I recall correctly jpeg specification doesn't specify anything about comment section contents.
Why would we need `FromString(string value)` method though?

Reply:
Sorry @br3aker I was confinced I'd replied to this!!

I'm just covering the most common use case, ASCII strings, in a nice manner. Saves using `String.AsMemory()` which feels a little clunky.